services/repository/src/main/java/org/sagebionetworks/repo/web/AccessRecordListener.java

```diff
@@ -0,0 +1,23 @@
+package org.sagebionetworks.repo.web;
+
+
+/**
+ * Abstraction for a listener to response accessRecord object.
+ * @author John
+ *
+ */
+public interface AccessRecordListener {
```

Comment:
AccessRecordDataListener. The description should be something meaningful. E.g. Listener used to provide additional information to an access record